.changeset/dry-hairs-move.md

```diff
@@ -0,0 +1,10 @@
+---
+"@ledgerhq/live-cli": major
+"ledger-live-desktop": major
+"live-mobile": major
+"@ledgerhq/live-common": major
+"@ledgerhq/cryptoassets": major
+"@ledgerhq/hw-app-icon": major
```

Comment:
⚠️ The  severity should definitely not be `major`.

Reply:
Hi @elbywan I have updated it, thanks

Reply:
Hey @shallcdn thanks, could you also please remove the old changeset file? ([.changeset/dry-hairs-move.md](https://github.com/LedgerHQ/ledger-live/pull/1677/files#diff-60ba29af1d5619063ccba773e74c3232b6d4d8255b69226827f49979cbcb9ec9))

Reply:
Thanks! 🙇